cumulus/parachains/runtimes/bridge-hubs/bridge-hub-westend/src/lib.rs

```diff
@@ -572,10 +591,15 @@ construct_runtime!(
 		BridgeRococoMessages: pallet_bridge_messages::<Instance1> = 44,
 		XcmOverBridgeHubRococo: pallet_xcm_bridge_hub::<Instance1> = 45,
 
-		EthereumInboundQueue: snowbridge_pallet_inbound_queue = 80,
-		EthereumOutboundQueue: snowbridge_pallet_outbound_queue = 81,
-		EthereumBeaconClient: snowbridge_pallet_ethereum_client = 82,
-		EthereumSystem: snowbridge_pallet_system = 83,
+		EthereumSystem: snowbridge_pallet_system = 80,
+		EthereumBeaconClient: snowbridge_pallet_ethereum_client = 81,
+		EthereumInboundQueue: snowbridge_pallet_inbound_queue = 82,
+		EthereumOutboundQueue: snowbridge_pallet_outbound_queue = 83,
```

Comment:
I would suggest not changing the order here, the pallet index should be consistent with https://github.com/polkadot-fellows/runtimes/blob/76d1fa680d00c3e447e40199e7b2250862ad4bfa/system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs#L593-L597 or some tests will fail.

Reply:
reverted a while ago